browser/css/jsdialogs.css

```diff
@@ -171,6 +171,12 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	margin: auto 0px;
 }
 
+#rightVBox.ui-grid-cell, #leftVBox.ui-grid-cell {
```

Comment:
can we use some unique prefix with the id of the dialog so we don't apply that rule unintended?

#PageNumberDialog i think it is

Reply:
and maybe a comment above for which dialog it is